modules/clickhouse/src/main/java/org/testcontainers/containers/ClickHouseContainer.java

```diff
@@ -0,0 +1,72 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.HttpWaitStrategy;
+
+import java.time.Duration;
+
+public class ClickHouseContainer<SELF extends ClickHouseContainer<SELF>> extends JdbcDatabaseContainer<SELF> {
```

Comment:
there is no need for self-typing here

Reply:
I don't get it. Almost all other JDBC containers have 'SELF', why this one shouldn't?

Reply:
Consider it a legacy :)